src/main/scala/wdl4s/values/WdlArray.scala

```diff
@@ -58,12 +58,12 @@ sealed abstract case class WdlArray(wdlType: WdlArrayType, value: Seq[WdlValue])
 
   def tsvSerialize: Try[String] = {
     wdlType.memberType match {
-      case t: WdlPrimitiveType => Success(value.map(_.valueString).mkString("\n"))
+      case t: WdlPrimitiveType => Success(value.map(_.valueString + "\n").mkString(""))
```

Comment:
`mkString` actually has a "suffix" parameter you can use for this.
Or, if you leave it like this, you don't need arguments at all, ie just `.mkString`